.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm test
```

Comment:
instead of the test script you should run lint since we don't do any testing

Reply:
when I changed test to run lint and then committed, I received this message in the console 

<img width="1288" alt="image" src="https://user-images.githubusercontent.com/39521158/177538791-c1789ccc-856c-4861-8106-d67f71fc9a05.png">